src/CoreWCF.Primitives/src/CoreWCF/IdentityModel/ReferenceList.cs

```diff
@@ -0,0 +1,140 @@
+using System.Collections.Generic;
+using CoreWCF.IdentityModel;
+using System.Runtime.CompilerServices;
+using System.Xml;
+using DictionaryManager = CoreWCF.IdentityModel.DictionaryManager;
+using ISecurityElement = CoreWCF.IdentityModel.ISecurityElement;
+using CoreWCF;
+using XD = CoreWCF.IdentityModel.XD;
```

Comment:
We really should consolidate the XD stuff into a single class. Can you open an issue for this?

Reply:
https://github.com/CoreWCF/CoreWCF/issues/254